jina/drivers/__init__.py

```diff
@@ -397,6 +399,162 @@ def _apply_all(
         """
 
 
+class DocsExtractUpdateMixin:
```

Comment:
This assumes a lot of parameters of the drivers, maybe it should be directly a Driver itself? And then they can directly inherit from it.

Reply:
> maybe it should be directly a Driver itself?

i thought about that, but the side-effect on making it default behavior is not clear to me and maybe an overshoot for this PR.